### ISSUE TITLE
Swagger not generated from Controllers in different assemblies with version 12.0.5 and NetCore22

### ISSUE THREAD

**Issue by YZahringer** [RSuter/NSwag#1806]
Controllers outside project specified in swaggerGenerator are ignored with v12.0.5 and NetCore22. I tried with NSwag.MSBuild and NSwagStudio. Same issue, there only generate swagger for controllers in main project.

It works well in v12.0.4 and NetCore21.

**Comment by RSuter** [comment]
Maybe thats because this changed in asp.net core?

**Comment by YZahringer** [comment]
@RSuter Yep, `services.AddMvc().AddApplicationPart(assembly);` solve the problem.

Documentation: https://docs.microsoft.com/en-us/aspnet/core/mvc/advanced/app-parts?view=aspnetcore-2.2

**Comment by RSuter** [comment]
I think this extension was already needed in 2.1, no?

**Comment by YZahringer** [comment]
No, but it may be a regression. It worked without it in 2.1.

In the documentation it is specified:
> By default MVC will search the dependency tree and find controllers (even in other assemblies).

It should work without `AddApplicationPart`.
